core/sodasql/cli/ingest.py

```diff
@@ -172,25 +173,125 @@ def flush_test_results(
         soda_server_client.scan_ended(start_scan_response["scanReference"])
 
 
-def resolve_artifacts_paths(
-    dbt_artifacts: Optional[Path] = None,
-    dbt_manifest: Optional[Path] = None,
-    dbt_run_results: Optional[Path] = None
-) -> Tuple[Path, Path]:
-    if dbt_artifacts:
-        dbt_manifest = Path(dbt_artifacts) / 'manifest.json'
-        dbt_run_results = Path(dbt_artifacts) / 'run_results.json'
-    elif dbt_manifest is None:
+def load_dbt_artifacts(
+    artifacts_dir: Path | None = None,
```

Comment:
Isn't this 3.10 only ? type checkers working < 3.10 will get confused with this .. right ?

Reply:
the `from __future__ import annotations` allows to do this in earlier versions